added example in javascript custom plugin (#7349)
Kong/docs.konghq.com: app/_src/gateway/plugin-development/pluginserver/javascript.md

directory that holds plugin source code also includes a `node_modules` directory.

Assuming plugins are stored under `/usr/local/kong/js-plugins`, the extra dependencies are
then defined in `/usr/local/kong/js-plugins/package.json`. 

Developers also need to run `npm install` under `/usr/local/kong/js-plugins` to install those dependencies locally
into `/usr/local/kong/js-plugins/node_modules`.

The Node.js version and architecture that runs the plugin server and
the one that runs `npm install` under plugins directory must match.

> When running TypeScript plugins `kong-pdk` needs to be defined as dependency in `package.json` like:

Reviewer: ```suggestion
When running TypeScript plugins, `kong-pdk` needs to be defined as a dependency in `package.json`:
```

````json
{
  "name": "ts_hello",
  "version": "0.1.0",
  "description": "hello TS plugin from kong",
  "main": "ts_hello.ts",
  "dependencies": {
    "kong-pdk": "^0.5.5"
  }
}
````

> and import it in your TypeScript file like:

Reviewer: ```suggestion
Then, import `kong-pdk` in your TypeScript file:
```

[This repository](https://github.com/Kong/kong-js-pdk/tree/master/examples)
contains examples of writing tests with `jest`.

## Example configuration

> Prepare the system installing the required dependencies. In Debian/Ubuntu based systems:

Reviewer: ```suggestion
Prepare the system by installing the required dependencies. In Debian/Ubuntu based systems:
```

````
apt update
apt install -y nodejs npm
npm install -g kong-pdk
````

> Copy the plugin code and the `package.json` file in `/usr/local/kong/js-plugins` and run:

Reviewer: ```suggestion
Copy the plugin code and the `package.json` file in `/usr/local/kong/js-plugins`, then run:
```

````
cd /usr/local/kong/js-plugins/ 
npm install
````

> To load plugins using the `kong.conf` [configuration file](/gateway/latest/production/kong-conf/), you have to map existing {{site.base_gateway}} properties to aspects of your plugin.  Here there is an example of loading plugin within `kong.conf`:

Reviewer: ```suggestion
To load plugins using the `kong.conf` [configuration file](/gateway/latest/production/kong-conf/), you have to map existing {{site.base_gateway}} properties to aspects of your plugin. Here is an example of loading a plugin within `kong.conf`:
```
